detect.go

```diff
@@ -210,6 +231,28 @@ func Detect(
 			})
 
 			requirements = append(requirements, packit.BuildPlanRequirement{
+				Name: "dotnet-core-aspnet-runtime",
+				Metadata: BuildPlanMetadata{
+					Launch: true,
+				},
```

Comment:
should this case request the specific version from the project file?

Reply:
I think that all version detection is moving to the `dotnet-core-aspnet-runtime` buildpack this ensures that a `runtimeconfig.json` will be present.